index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="title">
+      <a href="/"><img src="./image/panda_market.png" alt="판다마켓 로고" class="logo"></a>
+      <a href="./sign-in.html"><button class="login-button">로그인</button></a> 
+    </div>
+ 
+<div class="container0">
+    <!-- 텍스트와 사진 -->
+  <div class="inner_box0"> 
+    <div class text-box0>
+     <h1>일상의 모든 물건을 거래해 보세요</h1>
+     <a href="./item.html"><p class="see">구경하러가기</p></a>
+    </div>
+    <div class="contents0">
+       <img src="./image/Img_home_top.png">
+    </div>
+  </div> 
+</div>
+     <!-- main 컨테이너 -->
+ <div class="container1">
+  <div class="contents">
+   <img src="./image/Img_home_01.png"> 
+  </div>
+  <div class="text-box">
+    <p class="hot-item">Hot_item</p>
+    <h1>인기 상품을</h1>
+    <h1>확인해 보세요</h1>
+    <p class="description">가장 HOT한 중고거래 물품을 판다 마켓에서 확인해 보세요</p>
+  </div>
+ </div>
+    <!-- main 컨테이너 -->
+<div class="container1">
+     <div class="text-box2">
+        <h1>구매를 원하는 상품을 검색하세요</h1>
+        <p class="description">구매하고 싶은 물품은 검색해서 쉽게 찾아보세요</p>
+    </div>
+    <div class="contents">
+        <img src="./image/Img_home_02.png">
+     </div>   
+</div>
+    <!-- main 컨테이너 -->
+<div class="container1">
+    <div class="contents">
+        <img src="./image/Img_home_03.png">
+    </div>
+    <div class="text-box3">
+        <p class="register">register</p>
+        <h1>판매를 원하는<br> 상품을 등록하세요</h1>
+        <p class="description">어떤 물건이든 판매하고 싶은 상품을 쉽게 등록하세요</p>
+    </div>
+</div>
+
+ <div class="container2">
+      <!-- 텍스트와 사진  -->
+  <div class="inner_box">
+  <div text-box4>
+    <h2>믿을 수 있는<br> 판다마켓 중고 거래</h2>
+  </div> 
+    <div class="content-bottom">
+      <img src="./image/Img_home_bottom.png">
+    </div>
+  </div>
+</div>
+
+<footer class="footer">
+        <!-- 텍스트와 아이콘 -->
+      <div class="inner_box2">
+        <div class="copyright">
+           <p>©codeit - 2024</p>        
+        </div>
+          <div class="links">
+            <a href="./privacy_policy.html">privacy policy</a>
+            <a href="./FAQ.html">FAQ</a>
+          </div>
+          <div class="social_img">
+            <a href="https://www.facebook.com/" target="_blank" ><img src="./image/ic_facebook.png"></a>
+            <a href="https://x.com/?lang=ko" target="_blank"><img src="./image/Vector.png"></a>
+            <a href="https://www.youtube.com/" target="_blank"><img src="./image/Group.png"></a>
+            <a href="https://www.instagram.com/" target="_blank"><img src="./image/Vector (1).png"></a>
+          </div>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
          <div class="social_img">
            <li><a href="https://www.facebook.com/" target="_blank" ><img src="./image/ic_facebook.png"></a></li>
            <li><a href="https://x.com/?lang=ko" target="_blank"><img src="./image/Vector.png"></a></li>
            <li><a href="https://www.youtube.com/" target="_blank"><img src="./image/Group.png"></a></li>
            <li><a href="https://www.instagram.com/" target="_blank"><img src="./image/Vector (1).png"></a></li>
          </div>
```